### DISCUSSION TITLE
Centralized wifi reports

### DISCUSSION THREAD

**Discussion by bigdestroyer** [VREMSoftwareDevelopment/WiFiAnalyzer#427] [Ideas]
Good day

I have an idea for ISP that they can give compile their own app (for branding purposes) and that the clients can submit results to centralized web server. The ISP staff can access the web server to see the results that the client has picked up and rectify wifi settings as required.

Let me know if there is any unclarities.

**Comment by VREMSoftwareDevelopment** [comment]
This is a feature request, not a discussion
